src/components/Books.js

```diff
@@ -1,39 +1,48 @@
 import { useSelector, useDispatch } from 'react-redux';
 import { removeBook } from '../redux/books/books';
+import progressBar from '../images/progress_bar.png';
 
 const Books = () => {
-  const booksState = useSelector((state) => state.bookReducer);
+  const booksState = useSelector((state) => state.bookReducer.books);
+  const booksArray = Object.values(booksState);
   const dispatch = useDispatch();
 
   const remove = (id) => {
     dispatch(removeBook(id));
   };
 
   return (
-    <section>
-      {booksState.map((books) => (
-        <div key={`Book ${books.book.id}`} className="book-container">
+    <section className="books-section">
+      {booksArray.map((books, index) => (
+        <div key={Object.keys(booksState)[index]} className="book-card">
           <div className="book-info">
-            <h3 className="book-category">{books.book.category}</h3>
-            <h2 className="book-title">{books.book.title}</h2>
-            <h4 className="book-author">{books.book.author}</h4>
+            <h3 className="book-category">{books[0].category}</h3>
+            <h2 className="book-title">{books[0].title}</h2>
+            <h4 className="book-author">Author name</h4>
```

Comment:
- [ ] I am suggesting that you use your name instead of Author name which gives a more custom project look 😄.

Reply:
Thanks for the feedback. I have resolved them.